packages/opentelemetry-exporter-zipkin/src/types.ts

```diff
@@ -21,7 +21,7 @@ import * as api from '@opentelemetry/api';
  */
 export interface ExporterConfig {
   logger?: api.Logger;
-  serviceName: string;
+  serviceName?: string;
```

Comment:
How about we mention that it's completely optional to pass the config now. So the usage section will be something like:
"Install the exporter on your application and pass the options. It's optional to pass the options. If `serviceName` is omtted, it will be inferred from resource and in case not provided, serviceName will default to "OpenTelemetry Service". The `url` is also set to `http://localhost:9411/api/v2/spans` by default."